iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/normalize/AffMatchOrganizationNormalizer.java

```diff
@@ -29,44 +30,34 @@
     private StringNormalizer countryCodeNormalizer = new LenientComparisonStringNormalizer();
     
     private StringNormalizer websiteUrlNormalizer = new WebsiteUrlNormalizer();
-    
-    
+
     
     
     //------------------------ LOGIC --------------------------
     
-    
     /**
-     * Converts {@link Organization} into {@link AffMatchOrganization}
+     * Normalizes properties of <code>affMatchOrganization</code> with corresponding normalizers (see
+     * setters).<br/><br/>
+     * This method <b>changes the passed object and returns it</b>. It does NOT create a new one.
```

Comment:
The same question as for the affiliation normalizer. Is it safe to modify RDD elements?